src/main/java/com/autotune/analyzer/serviceObjects/CreateExperimentAPIObject.java

```diff
@@ -0,0 +1,206 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.analyzer.serviceObjects;
+
+import com.autotune.analyzer.kruizeObject.SloInfo;
+import com.autotune.analyzer.recommendations.ContainerRecommendations;
+import com.autotune.common.data.metrics.Metric;
+import com.autotune.analyzer.kruizeObject.RecommendationSettings;
+import com.autotune.common.k8sObjects.TrialSettings;
+import com.autotune.utils.KruizeConstants;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class CreateExperimentAPIObject extends BaseSO {
```

Comment:
Please add javadoc saying that this is simulating `KruizeObject` for the `createExperiment` API

Reply:
Done